### PR TITLE
i added my name because im testing git commit

### PR DESCRIPTION
**Pull request by tsilvaX**
JIRA Number: LUM-1

### CODE REVIEW

**Review by kotooriiii — COMMENTED** [review]
it's sheiks

File: SAMPLE_README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 **IMPORTANT: In order to use install this app, you need to be Whitelisted as a developer!test
 (https://overwolf.github.io/docs/start/sdk-introduction#get-whitelisted-as-a-developer)**
 
-# Sample - TS
+# Sample - TS thiago was here
```
Comment (kotooriiii) [inline SAMPLE_README.md:4]: test 2

**Review by kotooriiii — CHANGES_REQUESTED** [review]

File: SAMPLE_README.md (hunk 1 above)
Comment (tsilvaX) [reply SAMPLE_README.md:4]: this is actually ok, adnakwdwanda

**Review by kotooriiii — APPROVED** [review]
all good

**Pull request closed without merging**